downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,232 @@
+- The `lang.xml:strip()` function does not mutate an XML value. Therefore, using the `lang.xml:strip()` function on immutable XML values is allowed.
 
-#### Standard Library
-- 
+### Breaking Changes
 
-#### Developer Tools
+- An included record parameter of a function can only be specified after any required and/or defaultable parameters of the function.
+- Additive expressions and multiplicative expressions are no longer supported with numeric values when the static types of the operands belong to different numeric basic types.
+- Configurable variables are implicitly `final` now. Moreover, the type of such a variable is now effectively the intersection of the specified type and `readonly`. Therefore, configurable variables no longer support the `final` and `isolated` qualifiers.
+- Type narrowing will no longer take place for captured variables of an anonymous function since the narrowed type cannot be guaranteed during the execution of the function.
+- Type narrowing will now be reset after a compound assignment.
+- Worker message passing after waiting for the same worker has been disallowed.
+- When a named worker is used in a `wait` action, it can no longer be used in a variable reference anywhere else.
+- When the type-descriptor is ambiguous, it is parsed according to the following table in which the precedence is specified in decreasing order.
 
-##### Debugger
+  For example, `A & B | C` is considered to be `(A & B) | C`.
 
-- Added remote debugging support for the command, which runs the Ballerina executable JAR.
-- Added support for debugging single Ballerina tests in Visual Studio Code.
+  | Operator                  | Associativity |
+  |---------------------------|---------------|
+  | distinct T                |               |
+  | T[ ]                      |               |
+  | T1 & T2                   | left          |
+  | T1 &#124; T2              | left          |
+  | function (args) returns T | right         |
 
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+## Runtime Updates
+
+### Improvements
+
+#### Improved Configurable Variables to Support Enum Types
+
+Configurable variables with enum types are supported to provide values through command-line arguments and a TOML file.
+
+For example, see the configurable variable with the `enum` type.
+
+```ballerina
+public enum HttpVersion {
+    HTTP_1_1,
+    HTTP_2
+}
+
+configurable configLib:HttpVersion & readonly httpVersion = ?;
+```
+
+The value for `httpVersion` can be provided via the `Config.toml` file or as a command-line argument as below.
+
+**TOML:**
```

Comment:
Avoid using bold here as it will have more prominent view than the Level 4  heading above.

Reply:
Fixed.